docs/yul.rst

```diff
@@ -1015,3 +1015,54 @@ If you want to use Solidity in stand-alone Yul mode, you activate the optimizer
     solc --strict-assembly --optimize
 
 In Solidity mode, the Yul optimizer is activated together with the regular optimizer.
+
+Optimization step sequence
+--------------------------
+
+By default Yul optimizer applies its predefined sequence of optimization steps to the generated assembly.
+You can override this sequence and supply your own using the `--yul-optimizations` option when compiling in Solidity mode:
+
+.. code-block:: sh
+
+    solc --optimize --ir-optimized --yul-optimizations dhfoD[xarrscLMcCTU]uljmul
+
+By enclosing part of the sequence in square brackets (`[]`) you tell the optimizer to repeatedly apply that part until it no longer improves the size of the resulting assembly.
```

Comment:
```suggestion
By enclosing part of the sequence in square brackets (`[]`) you tell the optimizer to repeatedly
apply that part until it no longer improves the size of the resulting assembly.
```

Reply:
I was wrapping sentences on full stop (makes less noise in the diffs than manual wrapping when you need to update the text), but OK, I see now that the whole document is manually wrapped to a constant width.

Done.